docker-compose.elasticsearch.yaml

```diff
@@ -15,6 +15,11 @@ services:
       - VIRTUAL_HOST=$DDEV_HOSTNAME
       - HTTP_EXPOSE=9200:9200
       - HTTPS_EXPOSE=9201:9200
+      - "http.cors.allow-origin=*"
+      - "http.cors.enabled=true"
+      - "http.cors.allow-headers=X-Requested-With,X-Auth-Token,Content-Type,Content-Length,Authorization,Access-Control-Allow-Origin,Access-Control-Request-Headers"
+      - "http.cors.allow-credentials=true"
+      - "http.cors.allow-methods: OPTIONS, HEAD, GET, POST, PUT, DELETE"
```

Comment:
Are these environment variables? Or some other type of configuration? Dots in environment variables are unusual. And who consumes this information?

Reply:
This is configuration used by ES, based on documentation here: https://www.elastic.co/guide/en/elasticsearch/reference/current/behavioral-analytics-cors.html#behavioral-analytics-cors-enable-cors-elasticsearch